src/main/java/org/hydev/mcpm/client/commands/entries/RefreshController.java

```diff
@@ -0,0 +1,31 @@
+package org.hydev.mcpm.client.commands.entries;
+
+import org.hydev.mcpm.client.database.fetcher.DatabaseFetcher;
+import org.hydev.mcpm.client.database.fetcher.DatabaseFetcherListener;
+import org.hydev.mcpm.client.database.mirrors.MirrorSelectBoundary;
+
+import java.io.IOException;
+
+/**
+ * Controller for the refresh command
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-24
+ */
+public record RefreshController(DatabaseFetcher fetcher, DatabaseFetcherListener listener, MirrorSelectBoundary mirror)
```

Comment:
People don't like spacing things out? T_T

```suggestion
public record RefreshController(
    DatabaseFetcher fetcher,
    DatabaseFetcherListener listener,
    MirrorSelectBoundary mirror
)
```